core/shared/src/main/scala/org/threeten/bp/chrono/ChronoLocalDateImpl.scala

```diff
@@ -91,31 +91,31 @@ import org.threeten.bp.temporal.TemporalUnit
  *   the date type
  */
 @SerialVersionUID(6282433883239719096L)
-abstract class ChronoDateImpl[D <: ChronoLocalDate] private[chrono] ()
+abstract class ChronoLocalDateImpl[D <: ChronoLocalDate] private[chrono] ()
```

Comment:
Why the name change?

Reply:
For some reason, Scala Native wouldn't compile without this change.